source/wp-content/themes/wporg-parent-2021/theme.json

```diff
@@ -340,7 +340,7 @@
 			},
 			"layout": {
 				"contentSize": "680px",
-				"wideSize": "1160px",
+				"wideSize": "1280px",
```

Comment:
This is the minimum width required for the buttons to be able to sit next to each other on desktop with the current padding of the buttons and columns. It may have wider effects we're not happy with though.

Reply:
It's because this text has the `6.0.1` while the mockup has `6.0` — leave this size as-is, and we can play with the padding and such on the site itself. The important part of this pattern is the two column layout, the side-by-side buttons can be managed once the pattern is used.